roles-in-tech/machine-learning.md

```diff
@@ -36,10 +39,35 @@
 
 ### Materials
 
+- [Example video (10 min)](https://example.com)
+
```

Comment:
Forget removing the link; please put your prerequisites in the master version of the file. This is the wrong version.